Cargo.toml

```diff
@@ -8,6 +9,11 @@ ed25519-zebra = "3.1"
 rand = "0.8"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
-ucan = "0.2.0"
-ucan-key-support = "0.1.3"
+ucan = "0.4.0"
```

Comment:
this needs to stay at 0.2.0 for the moment - once @icidasset is ready to upgrade ts-odd to ucan 0.10, we can make the switch, but the `fct` and `prf` fields at a minimum are mutually incompatible and no compatibility layer was built.